app/scripts/hglib.js

```diff
@@ -102,3 +102,53 @@ export const viewer = (element, viewConfig, options) => {
 
   return hg.api;
 };
+
+export const trackViewer = (element, [xMin, xMax, yMin, yMax], trackConfig, options) => {
+  if (!trackConfig.options.colorbarPosition) {
+    trackConfig.options.colorbarPosition = 'hidden';
+    // If the colorbar were all SVG, I could change the colorbar Z position,
+    // and move the whole thing around, but apart from the handlebars,
+    // I think it's all part of the canvas, right?
+  }
+  if (!trackConfig.options.labelTextOpacity) {
+    trackConfig.options.labelTextOpacity = 0;
+    // TODO: There is still a rectangle in the lower right... Is there an easy way to just turn
```

Comment:
You should be able to set `labelPosition` to `hidden`. You could also set `labelBackgroundOpacity` to 0.